src/Components/Caesar/Caesar.js

```diff
@@ -75,4 +74,36 @@ class Caesar extends Component {
 
 export default Caesar;
 
-// <CaesarButtons offset={this.state.offset} onOffsetChange={this.handleOffsetChange} />
\ No newline at end of file
+// <CaesarButtons offset={this.state.offset} onOffsetChange={this.handleOffsetChange} />

```

Comment:
If you aren't going to use your commented-out code, then it might be wise to remove it altogether - especially when the commented-out code exists in your Git history.